mdn-drafts/ScreenAdvanced.md

```diff
@@ -0,0 +1,104 @@
+---
+recipe: api-interface
+title: 'ScreenAdvanced'
+mdn_url: /en-US/docs/Web/API/ScreenAdvanced
+specifications: https://webscreens.github.io/window-placement/#screenadvanced
+browser_compatibility: api.ScreenAdvanced
+---
+
+## Description
+
+The `ScreenAdvanced` interface of the Window Placement API extends the
+`Screen` interface and provides additional per-screen information.
+
+The `ScreenAdvanced` interface is a child of `Screen` and inherits its members.
+
+## Constructor
+
+None.
+
+## Properties
+
+_Inherits properties from its parent, `Screen`._
+
+**`ScreenAdvanced.left`**
+
+Returns the distance in pixels to the left edge of the screen from the primary screen origin.
+
+**`ScreenAdvanced.top`**
+
+Returns the distance in pixels to the top edge of the screen from the primary screen origin.
+
+**`ScreenAdvanced.isPrimary`**
+
+Returns true if this screen is the primary screen.
+
+**`ScreenAdvanced.isInternal`**
+
+Returns true if this scren is built into the device, like a laptop display.
+
+**`ScreenAdvanced.devicePixelRatio`**
+
+Returns ratio of this screen's resolution in physical pixels to its resolution in CSS pixels.
+
+**`ScreenAdvanced.id`**
+
+Returns a temporary, generated per-origin unique ID; resets when cookies are deleted.
+
+**`ScreenAdvanced.label`**
+
+Returns a user-friendly label string for the screen, determined by the user agent and OS.
+
+**`ScreenAdvanced.pointerTypes`**
+
+Returns an array of `PointerType` strings.
```

Comment:
optional: ..."strings, representing input modes supported by the display."

Reply:
Added.